src/styles/_typography.scss

```diff
@@ -28,27 +17,43 @@ button {
 }
 
 h1 {
-  font-size: 2.1rem;
+  font-size: $font-size-largest;
+  font-weight: $font-weight-regular;
 }
 
 h2 {
-  font-size: 1.65rem;
+  font-size: $font-size-extra-large;
+  font-weight: $font-weight-bold;
 }
 
 h3 {
-  font-size: 1.25rem;
+  font-size: $font-size-large;
+  font-weight: $font-weight-bold;
+}
+
+h4 {
+  font-size: $font-size-medium;
+  font-weight: $font-weight-regular;
+}
+
+h5 {
+  font-size: $font-size-small;
+  font-weight: $font-weight-regular;
+}
+
+h6 {
+  font-size: $font-size-extra-small;
+  font-weight: $font-weight-regular;
 }
 
 @media screen and (min-width: $tablet-breakpoint) {
-  // increases font-size of all items by 10%
   :root {
     font-size: 110%;
```

Comment:
What is this? Can you chat to me about this? Not sure I understand what is happening here?

Reply:
This code was already in the project. What it does is once the screen size goes above the breakpoint, which here is 768px, all font sizes, across the whole app, increase by 10% from the original size.